Update contributing doc to add uploading data files to Zenodo info (#455)
astropy/astropy-tutorials: CONTRIBUTING.rst

want to include them in the ``astropy/astropy-tutorials`` git repository, as
that will drastically slow down cloning the repository. Instead, we encourage
use of the `astropy.utils.download_files` function, and will host data files on
the `<http://data.astropy.org>`_ server, or to use Zenodo to host the data files.
To do the former, use the following
procedure:

> 

Reviewer: Change this wording so that "We encourage writers to host data files on Zenodo, using the following procedure."

....

"In the past, we have used the `astropy.utils.download_files` function in conjunction with storing files on the `<http://data.astropy.org>`_ server. This requires the following procedure:"

  If you do this, the only change necessary when merging your notebook will be to set ``tutorialpath`` to  ``'http://data.astropy.org/tutorials/My-tutorial-name/'``.

> Using Zenodo to host large data files is very straightforward and involves far less steps. To use

Reviewer: Modify this introduction to coincide with intro, above.

this approach do the following:

* You will need to sign up for an account with Zenodo before you can upload a file there.

* After a Zenodo account has been properly set up, be sure to log in and do a new upload. Follow
the Zenodo instructions and complete all the required fields as the bare minimum in order to have
the data file(s) uploaded to their records. Once this is done you will have a link to share the data.

> * With the link which has the format ``https://zenodo.org/api/records/:id``, and the HTTP GET request

Reviewer: ```suggestion
* You Zenodo file link will have the format ``https://zenodo.org/api/records/:id``. Use the HTTP GET request
```

> needed to retrieve the record using the Python package ``requests`` is as below::

Reviewer: ```suggestion
to retrieve the data, using the Python package ``requests`` as shown below::
```

    import requests
    r = requests.get("https://zenodo.org/api/records/1234)

> Basically to use the output you simply need to add the following lines to your Jupyter notebook::

Reviewer: ```suggestion
To open the data file, you can add the following lines to your Jupyter notebook::
```

  If you do this, the only change necessary when merging your notebook will be to set ``tutorialpath`` to  ``'http://data.astropy.org/tutorials/My-tutorial-name/'``.

For larger data files that are bigger than 10 MB in size, we recommend hosting with Zenodo. Using Zenodo
to host large data files is very straightforward and involves potentially far less steps. To use
this approach do the following:

* You will need to sign up for an account with Zenodo before you can upload a file there.

* After a Zenodo account has been properly set up, be sure to log in and do a new upload. Follow
the Zenodo instructions and complete all the required fields as the bare minimum in order to have
the data file(s) uploaded to their records. Once this is done you will have a link to share the data.

> * With the link which has the format ``https://zenodo.org/api/records/:id``, and the HTTP GET request
> needed to retrieve the record using the Python package ``requests`` is as below::

Reviewer 2: ```suggestion
* With the link to the data file record, which has the format ``https://zenodo.org/api/records/:id``, an example HTTP GET request needed to retrieve the data using the Python package ``requests`` is shown below::
```

    import requests
    r = requests.get("https://zenodo.org/api/records/1234)

> Basically to use the output you simply need to add the following lines to your Jupyter notebook::

Reviewer 2: ```suggestion
To use the output as a locally stored file, you would first need to write the file contents to a file, for example::
```
